### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because the build is broken.

### CODE REVIEW

File: src/Admin/BaseMediaAdmin.php [hunk 1]
```diff
@@ -90,7 +103,8 @@ public function configurePersistentParameters(): array
         $categoryId = $this->getRequest()->get('category');
 
         if (null !== $this->categoryManager && !$categoryId) {
-            $category = $this->categoryManager->getRootCategory($context);
+            $categories = $this->categoryManager->getRootCategoriesForContext($this->contextManager->find($context));
+            $category = $categories[0];
```
Comment (jordisala1991) [inline src/Admin/BaseMediaAdmin.php:107]: I don't like a lot having to do this kind of access to the array, but previously this returned one root category, now returns all (whatever that means). Same for the other places.

Comment (VincentLanglet) [reply src/Admin/BaseMediaAdmin.php:107]: Category are set this way
```
$this->categories[$context->getId()][$category->getId()] = $category;
```
So I'm not sure `[0]` is the right way to do it.
`current($categories)` maybe instead ?

File: src/Controller/MediaAdminController.php [hunk 2]
```diff
@@ -70,8 +72,12 @@ public function listAction(Request $request): Response
...
 
         $rootCategory = null;
-        if ($this->has('sonata.media.manager.category')) {
-            $rootCategory = $this->get('sonata.media.manager.category')->getRootCategory($context);
+        if ($this->has('sonata.media.manager.category') && $this->has('sonata.media.manager.context')) {
+            $rootCategories = $this->get('sonata.media.manager.category')->getRootCategoriesForContext(
+                $this->get('sonata.media.manager.context')->find($context)
+            );
+
+            $rootCategory = $rootCategories[0];
```
Comment (VincentLanglet) [inline src/Controller/MediaAdminController.php:80]: Same, `current($rootCategories)` is better I think

File: src/Admin/BaseMediaAdmin.php (hunk 1 above)
Comment (jordisala1991) [reply src/Admin/BaseMediaAdmin.php]: Updated.

File: src/Controller/MediaAdminController.php (hunk 2 above)
Comment (jordisala1991) [reply src/Controller/MediaAdminController.php]: Fixed.

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**